### DISCUSSION TITLE
backup folder

### DISCUSSION THREAD

**Discussion by benutzer74** [seedvault-app/seedvault#193] [Q&A]
Hello,
I created a backup on the external sd card, there is now a folder named .Seedvault is this the place where backup was stored?
Can I now move that hidden folder to my PC and when I need to restore data move it back from PC to sdcard and restore from there?
Thanks

**Comment by lazytownfan** [comment]
I think so (if you don't need to take continuous "snapshots" of Seedvault), though others should correct me if this is wrong.
